MANIFEST.SKIP

```diff
@@ -8,6 +8,7 @@
 ^\.github/pull_request_template\.md$
 ^\.travis\.yml$
 ^docs/internal-documentation/
+^lib/auto/share/dist/
```

Comment:
Shouldn't we be including this?

Reply:
There is another solution in Engine. I suggest that we have the same solution in all repositories where CPAN packages are created. Unless there is a good reason against that.